### PR TITLE
fix cloudbuild configs

### PR DESCRIPTION
**Pull request by BenTheElder**
follow up https://github.com/kubernetes-sigs/kind/pull/3125

entrypoint is a single string, need to split to args.

### CODE REVIEW

File: images/base/cloudbuild.yaml [hunk 1]
```diff
@@ -3,5 +3,5 @@ options:
   substitution_option: ALLOW_LOOSE
 steps:
 - name: gcr.io/k8s-testimages/krte:latest-master
-  env:
-  entrypoint: make -C images/base push
+  entrypoint: make
+  args: ['-C' 'images/base' 'push']
```
Comment (aojea) [inline images/base/cloudbuild.yaml:7]: don't you need a comma?

Comment (aojea) [reply images/base/cloudbuild.yaml:7]: it seems you do https://cloud.google.com/build/docs/configuring-builds/create-basic-configuration

Comment (BenTheElder) [reply images/base/cloudbuild.yaml]: yep

**Comment by BenTheElder** [comment]
https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-kind-push-base-image/1636158171183583232 🎉 

gcr.io/k8s-staging-kind/base:v20230316-a681949f@sha256:d6ccc95bab9972b302016790e9eb23ec25ccfb3a9d352ab9115802ec8c4c7fdd

**Comment by BenTheElder** [comment]
two still failed:
https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-kind-push-local-path-provisioner-image/1636158171330383872
https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-kind-push-kindnetd-image/1636158171233914880

I think they're timing out because the slow cross-builds. we can bump the machine size or timeout limit, or we can get to removing those.

**Comment by BenTheElder** [comment]
the rest passed.

**Comment by aojea** [comment]
timed out in one hour?

**Pull request merged**